test/alternator/test_table.py

```diff
@@ -553,3 +554,59 @@ def creates():
                     time.sleep(1)
                     continue
                 raise
+
+# Test that DeleteTable returns correct TableDescription (issue #11472)
+def test_delete_table_description(dynamodb):
+    table_name = unique_table_name()
+
+    table = create_table(dynamodb, table_name)
+    table.meta.client.get_waiter('table_exists').wait(TableName=table_name)
+    got = table.delete()['TableDescription']
+    table.meta.client.get_waiter('table_not_exists').wait(TableName=table_name)
+
+    assert got['TableName'] == table_name
+    assert got['TableStatus'] == 'DELETING'
+
+    # as far as we still return incorrect CreationDateTime we check only field existence (issue #5013)
+    # when fix #5013 add CreationDateTime value checking here
+    assert 'CreationDateTime' in got
+
+    assert 'TableId' in got
+    assert fullmatch('[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}', got['TableId'])
+
+    assert 'TableArn' in got and got['TableArn'].startswith('arn:')
+
+    expected_schema = {
+        'KeySchema': [ { 'AttributeName': 'p', 'KeyType': 'HASH' },
+                    { 'AttributeName': 'c', 'KeyType': 'RANGE' }
+        ],
+        'AttributeDefinitions': [
+                    { 'AttributeName': 'p', 'AttributeType': 'S' },
+                    { 'AttributeName': 'c', 'AttributeType': 'S' },
+        ]
+    }
+    assert got['KeySchema'] == expected_schema['KeySchema']
+    # The list of attribute definitions may be arbitrarily reordered
+    assert multiset(got['AttributeDefinitions']) == multiset(expected_schema['AttributeDefinitions'])
+
+    assert got['BillingModeSummary']['BillingMode'] == 'PAY_PER_REQUEST'
+    assert 'LastUpdateToPayPerRequestDateTime' in got['BillingModeSummary']
+    assert got['BillingModeSummary']['LastUpdateToPayPerRequestDateTime'] == got['CreationDateTime']
+
+    assert got['ProvisionedThroughput']['NumberOfDecreasesToday'] == 0
+    assert got['ProvisionedThroughput']['WriteCapacityUnits'] == 0
+    assert got['ProvisionedThroughput']['ReadCapacityUnits'] == 0
+
+# Test that DeleteTable returns correct TableDescription (issue #11472) and has no not implemented fields (see #5026 issue)
+# after any field implementation move its testing code to the 'test_delete_table_description' test
+@pytest.mark.xfail(strict=True, reason="#5026: TableDescription still doesn't implement these fields")
```

Comment:
Please remove this "strict=True". I want to be able to run this test on AWS DynamoDB, and see it XPASS, not FAIL. 
Also, If one day we fix this bug in Alternator and forget to remove this xfail, we will later notice an XPASS and can remove the xfail - it's not a good reason (I think) to fail the test.

Reply:
Done